src/app/ApplicationTemplate.Shared.Views/Framework/Connectivity/Connectivity.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.Networking.Connectivity;
+
+namespace ApplicationTemplate;
+
+public sealed class Connectivity : IConnectivity, IDisposable
+{
+	public Connectivity()
+	{
+		NetworkInformation.NetworkStatusChanged += OnNetworkStatusChanged;
```

Comment:
You could probably avoid subscribing until someone subscribes to `ConnectivityChanged`.
This can be refactored in a later PR.